proto/greenfield/sp/events.proto

```diff
@@ -44,6 +46,8 @@ message EventEditStorageProvider {
   string approval_address = 5 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   // gc_address defines one of the storage provider's accounts which is used for gc purpose
   string gc_address = 6 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // seal_bls_key defines the bls pub key owned by storage provider used when sealing object
+  string seal_bls_key = 7;
```

Comment:
Need to add id to the event. For other events, we may also need to add `id` field.

Reply:
@alexgao001 ditto